src/api/script/start_test_backend

```diff
@@ -291,6 +291,8 @@ ActiveRecord::Base.transaction(requires_new: true) do
   Package.all.each{ |pkg| pkg.sources_changed }
   UpdatePackageMetaJob.new.perform
 end
+Project.find_by_name('BaseDistro').update_product_autopackages
```

Comment:
because some lines before the product source files got committed. The backend has created further package containers and the api needs to follow